### PR TITLE
1ESPT conversion

### PR DESCRIPTION
**Pull request by mmitche**

### CODE REVIEW

**Review by premun — APPROVED** [review]

**Comment by mmitche** [comment]
`tsaoptions.json` will also need an area path update, but I don't know what the target area path should be at the moment.

**Comment by mmitche** [comment]
Test build: https://dev.azure.com/dnceng/internal/_build/results?buildId=2603412&view=results

**Comment by premun** [comment]
@mmitche in ServiceTree this repo is under Product Constructions Service. So possibly it should match. But we should ultimately put this on the product team

File: .config/tsaoptions.json
Comment (Prince-GH) [inline .config/tsaoptions.json:1]: may i know what you have updated in this json file

Comment (mmitche) [reply .config/tsaoptions.json:1]: It's a target path for where bugs get auto-filed when some some internal tooling finds issues.

Comment (Prince-GH) [reply .config/tsaoptions.json:1]: OKey

**Pull request merged**